src/apps/hector-network/binance-smart-chain/hector-network.bsc-bond.contract-position-balance-fetcher.ts

```diff
@@ -0,0 +1,59 @@
+import { Inject } from '@nestjs/common';
```

Comment:
This file is no longer necessary (the balances will be resolved using `src/apps/hector-network/binance-smart-chain/hector-network.bsc-bond.contract-position-fetcher.ts`)